crates/amaru/src/bin/amaru/cmd/import_headers.rs

```diff
@@ -82,7 +82,7 @@ pub(crate) async fn import_headers_for_network(
     config_dir: &Path,
     chain_dir: &PathBuf,
 ) -> Result<(), Box<dyn Error>> {
-    let db = RocksDBStore::new(RocksDbConfig::new(chain_dir.into()))?;
+    let db = RocksDBStore::unsafe_open(RocksDbConfig::new(chain_dir.into()))?;
```

Comment:
`unsafe` in Rust suggests unsafe memory blocks ( https://doc.rust-lang.org/book/ch20-01-unsafe-rust.html ). I would suggest calling this `open_and_migrate`?